### DISCUSSION TITLE
Unique contraint is not working as expected

### DISCUSSION THREAD

**Discussion by Tamimjabr** [prisma/prisma#22651] [Q&A]
### Bug description

I am trying to add two regions:
1. Habo
2. Håbo
Prisma is failing because of unique contraint on name (I am generating two different slugs)

### How to reproduce

Schema

model Region {
  id                Int                @id @default(autoincrement())
  name              String             @unique  @db.VarChar(255) 
  slug              String             @unique @db.VarChar(255)
  countryId         String
  type              RegionType         @default(CITY)
  createdAt         DateTime           @default(now())
  updatedAt         DateTime           @updatedAt
  country           Country            @relation(fields: [countryId], references: [code], onDelete: Cascade)
  jobs              Job[]
  companies         Company[]
  mailSubscriptions MailSubscription[]
  parentRegions     Region[]           @relation("RegionToRegion")
  subRegions        Region[]           @relation("RegionToRegion")
  lat               Float?             @db.Float
  lon               Float?             @db.Float

  @@index([countryId])
}

### Expected behavior

records should be added because they are two different names

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version
v5.8.0

**Comment by SevInf** [comment]
That's not a prisma issue - we don't enforce unique constraints, database does. Databases have a thing called unicode collation settings, that determine how strings are sorted and compared, whether or not case and diacritics matter and so on. 

If you want `Habo` and `Håbo` to be treated differently, you have to change those settings. 
You did not specify the database you are using, so I can not point you to the right instructions, unfortinately.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
